app/middlewares/user.js

```diff
@@ -0,0 +1,20 @@
+const validator = require('validator');
```

Comment:
why this should be in a middleware? would you use it again in another endpoints?

Reply:
Yes, I have to use it again in LogIn mutation